ui/src/copyable_text.tsx

```diff
@@ -51,11 +51,13 @@ export interface CopyableText {
   name?: S
   /** True if the component should allow multi-line text entry. */
   multiline?: B
+  /** The adjustable height of the textbox. */
```

Comment:
```suggestion
  /** Custom height in px, e.g. '200px'. Requires `multiline` to be set. */
```

Don't forget to run `make generate` after this change.